packages/utils/src/bulk-insert-patients.ts

```diff
@@ -54,38 +54,77 @@ function sleep(ms: number) {
   return new Promise(resolve => setTimeout(resolve, ms));
 }
 
+function toTitleCase(str: string): string {
+  return str
+    .toLowerCase()
+    .split(" ")
+    .map(s => s.charAt(0).toUpperCase() + s.substring(1))
+    .join(" ");
+}
+
+function normalizeGender(gender: string): "M" | "F" {
+  const lowerGender = gender.toLowerCase();
+  if (lowerGender === "male" || lowerGender === "m") {
+    return "M";
+  } else if (lowerGender === "female" || lowerGender === "f") {
+    return "F";
+  }
+  throw new Error(`Invalid gender ${gender}`);
+}
+
+function normalizeName(name: string): string {
+  return toTitleCase(name);
+}
+
+function normalizeAddressLine(addressLine: string): string {
+  return toTitleCase(addressLine);
+}
+
+function normalizeCity(city: string): string {
+  return toTitleCase(city);
+}
+
+function normalizeEmail(email: string): string {
+  return email.toLowerCase();
+}
+
+function normalizeState(state: string): USState {
+  if (
+    state.length === 2 &&
+    Object.values(states).includes(USState[state as keyof typeof USState])
```

Comment:
NABD b/c `USState[..]` contains the 2 letters for states, but we could simplify this by just checking `if (Object.values(states).includes(state))`, right?

Reply:
yes good point